modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ManagementToolbar.tsx

```diff
@@ -12,38 +12,34 @@
  * details.
  */
 
-import ClayButton from '@clayui/button';
-import ClayIcon from '@clayui/icon';
-import ClayManagementToolbar from '@clayui/management-toolbar';
-import {useModal} from '@clayui/modal';
 import {
 	API,
-	ModalEditExternalReferenceCode,
+	ManagementToolbar as GenericManagementToolbar,
```

Comment:
Hey @naraandrade, what do you think instead of renaming this imported component we can rename this whole component to `ObjectManagementToolbar` making it more specific and keeping the name of the imported component?